src/frontend/src/pages/GanttPage/GanttPageFilter.tsx

```diff
@@ -0,0 +1,188 @@
+}
+
+const GanttPageFilter: FC<GanttPageFilterProps> = ({
+  car1Handler,
+  car2Handler,
+  status,
+  statusHandler,
+  teamHandler,
+  startHandler,
+  endHandler,
+  expandedHandler,
+  teamList,
+  selectedTeam,
+  currentStart,
+  currentEnd,
+  resetHandler
+}) => {
+  const carFilters = (
+    <Grid item container direction="column" xs={12} md={1} sx={{ justifyContent: 'start', alignItems: 'start' }}>
+      <Grid item xs={12} md={1}>
+        Car
+      </Grid>
+      <Grid item container xs={12} md={1}>
+        <Grid item>
+          <Checkbox
+            defaultChecked
+            icon={<LooksOneOutlinedIcon />}
+            checkedIcon={<LooksOneOutlinedIcon />}
+            onChange={car1Handler}
+            sx={{
+              color: 'white',
+              '&.Mui-checked': {
+                color: '#ef4345'
+              },
+              '& .MuiSvgIcon-root': {
+                fontSize: '2rem'
+              },
+              paddingLeft: 0
+            }}
+          />
+        </Grid>
+        <Grid item>
+          <Checkbox
+            defaultChecked
+            icon={<LooksTwoOutlinedIcon />}
+            checkedIcon={<LooksTwoOutlinedIcon />}
+            onChange={car2Handler}
+            sx={{
+              color: 'white',
+              '&.Mui-checked': {
+                color: '#ef4345'
+              },
+              '& .MuiSvgIcon-root': {
+                fontSize: '2rem'
+              },
+              paddingLeft: 0
+            }}
+          />
+        </Grid>
+      </Grid>
+    </Grid>
+  );
+
+  const buttons = (
+    <Grid
+      item
+      container
+      xs={12}
+      md={3}
+      spacing={1}
+      sx={{ justifyContent: 'end', alignItems: 'center', alignSelf: 'center', justifySelf: 'end' }}
+    >
+      <Grid item>
+        <Button
+          onClick={() => {
+            expandedHandler(true);
+          }}
+          startIcon={<UnfoldMoreIcon />}
+        >
+          Expand
+        </Button>
+      </Grid>
+      <Grid item>
+        <Button
+          onClick={() => {
+            expandedHandler(false);
+          }}
+          startIcon={<UnfoldLessIcon />}
+        >
+          Collapse
+        </Button>
+      </Grid>
+      <Grid item>
+        <Button onClick={resetHandler} startIcon={<RestartAltIcon />}>
+          Reset
+        </Button>
+      </Grid>
+    </Grid>
+  );
+
+  return (
+    <PageBlock title="Filters" style={{ flexGrow: 1 }}>
+      <Grid container rowSpacing={1} columnSpacing={1} sx={{ justifyContent: 'start', alignItems: 'start' }}>
+        {carFilters}
+        <Grid item xs={12} md={2}>
+          <FormControl sx={{ width: '100%' }}>
+            <FormLabel>Status</FormLabel>
+            <Select value={status} onChange={statusHandler}>
+              <MenuItem value="All Statuses">All Statuses</MenuItem>
+              {Object.values(WbsElementStatus).map((status) => {
+                return <MenuItem value={status}>{status}</MenuItem>;
```

Comment:
Each child in a list should have a unique "key" prop. (appears at least again on line 152 and maybe somewhere else) 
ik its pretty insignificant but i just did the ticket to fix these little bugs and might as well do it now